spotinst_sdk2/clients/elastigroup/__init__.py

```diff
@@ -1461,12 +1468,12 @@ def create_elastigroup(self, group):
         """
         group = gcp_elastigroup.ElastigroupCreationRequest(group)
 
-        excluded_group_dict = self.exclude_missing(json.loads(group.toJSON()))
+        excluded_missing_dict = self.exclude_missing(json.loads(group.toJSON()))
```

Comment:
excluded_group_dict and formatted_group_dict are ok, revert them back throughout.